generators/spring-data-relational/command.ts

```diff
@@ -26,6 +27,7 @@ const command: JHipsterCommandDefinition = {
       scope: 'storage',
     },
   },
+  import: [GENERATOR_SERVER],
```

Comment:
This can be removed.
Is used to build cli options only.
Server imports from relational:
https://github.com/jhipster/generator-jhipster/blob/dc5d78c1810f1edd9ee6cc2ff9b7b4236f802169/generators/server/command.ts#L171

Reply:
```suggestion
```